pkg/game/board.go

```diff
@@ -297,6 +297,7 @@ func (board *board) roadCanBePlaced(checkedTile elements.PlacedTile, checkedRoad
 			neighbourTile,
 			neighbourRoad.Feature,
 			true,
+			false,
```

Comment:
Annotate arguments passed to the function so it is easier to follow

Reply:
actually argument removed